code/modules/clothing/head/misc_special.dm

```diff
@@ -184,6 +184,26 @@
 /obj/item/clothing/head/kitty/cursed
 	canremove = 0
 
+/obj/item/clothing/head/kitty/equipped(mob/M, var/slot)
+	..()
+	var/mob/living/carbon/human/H = M
+	if(!istype(H))
+		return
+	if(H.head == src)
+		to_chat(H, "<span class='sinister'>Something on your head is making you feel a little lightheaded...</span>")
+
+/obj/item/clothing/head/kitty/unequipped(mob/living/carbon/human/user, var/from_slot = null)
+	..()
+	if(from_slot == slot_head && istype(user))
+		to_chat(user, "<span class='info'>Your head starts to feel better again.</span>")
+
+/obj/item/clothing/head/kitty/OnMobLife(var/mob/living/carbon/human/wearer)
+	if(!istype(wearer))
+		return
+	if(wearer.head == src)
```

Comment:
Use `get_item_by_slot(slot_head)` instead of reading the inventory variables directly.